### PR TITLE
Add sync/async operators to Authorize step

### PR DESCRIPTION
**Pull request by maikmb**
Fixes and changes #

## Proposed Changes

1. Add sync/async operators to authorize step
2. Rename tests files to `examples/todoapp`

### CODE REVIEW

**Comment by jhomarolo** [comment]
Hi Maik, could you please, update the documentation?

File: examples/todoapp/usecases/listItems.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const listItems = (injection) =>
 
         request: { listId: Number },
 
-        authorize: (user) => user.isAdmin ? Ok() : Err(),
+        authorize: async (user) => Promise.resolve(user.isAdmin ? Ok() : Err()),
```
Comment (jhomarolo) [inline examples/todoapp/usecases/listItems.js:13]: Why use a promise resolve here?

**Review by jhomarolo — CHANGES_REQUESTED** [review]
Hi @maikmb, could you please, update the documentation?

File: examples/todoapp/usecases/listItems.js (hunk 1 above)
Comment (maikvortx) [reply examples/todoapp/usecases/listItems.js]: I forgot to remove this code. Adjusted!

**Pull request merged**